charts/rest-devicemanager/templates/middleware-forward-auth.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: "{{ include ".helm.fullname" . }}-forward-auth"
+  namespace: {{ .Values.namespace }}
+spec:
+  forwardAuth:
+    address: http://baseline-oathkeeper-api:4456/decisions
```

Comment:
@ak-slalom this is problematic... I think we do this elsewhere but once we decouple the auth-service from baseline, this name is going to change... @davenicoll-slalom

Reply:
@ak-slalom , when we are going to plan activity. No value should be hardcoded

Thank You
Kumar Yendamuri

Reply:
swap baseline with auth-service as far as I can tell (or w/e the release name is... auth-service[s]). The chart uses release.name to generate its names. I will deploy to CI today and see what the value is, but we should come up with an override for this so we can change it easily across helm charts if the name changes for any reason.

Reply:
aren't you still using baselines-ory? I created a whole different set of helm charts for this so we can deprecate the rest

Reply:
Waiting for this PR to be merged.
Do you have an upcoming work schedule?

Reply:
replace `baseline-` with `auth-service-` and we should be good for when I merge my massive PR

Reply:
I changed from `baseline-` to `auth-service-` !